### ISSUE TITLE
Switch to python/fabric based installation method for PnF at SPS (Trac #213)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#213]
Rethink the deployment process used for PnF software releases at pole.  The current system used in EL4 OS is very manual.  

Investigate the python Fabric tool.

Migrated from https://code.icecube.wisc.edu/ticket/213
```json
{
    "status": "closed", 
    "changetime": "2019-01-11T19:24:41", 
    "description": "Rethink the deployment process used for PnF software releases at pole.  The current system used in EL4 OS is very manual.  \n\nInvestigate the python Fabric tool.", 
    "reporter": "blaufuss", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1547234681920543", 
    "component": "jeb + pnf", 
    "summary": "Switch to python/fabric based installation method for PnF at SPS", 
    "priority": "normal", 
    "keywords": "", 
    "time": "2010-12-01T15:36:42", 
    "milestone": "", 
    "owner": "blaufuss", 
    "type": "enhancement"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20101201T15:36:58: 
* blaufuss changed status from "new" to "closed"
* blaufuss changed resolution from "" to "fixed"

**Comment by icecube-trac** [comment]
Trac update at 20101201T15:38:34: blaufuss commented:

> This is done:  
> 
> http://code.icecube.wisc.edu/svn/projects/pnf_setup/trunk
> 
> Has a fabric file and all files needed to complete this. Tested at SPTS and ready for use
> at new sps-access.
> 
> Docs are:
> 
> http://wiki.icecube.wisc.edu/index.php/Deploying_PnF_At_SPS

**Comment by icecube-trac** [comment]
Trac update at 20190111T18:58:04: nwhitehorn commented:

> In changeset 167915:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="167915"
> Fixes #213.   Buggy test was written to chase down another bug in 2005, see 
> 
>   root-icetray/trunk/private/test/ServicesAtDestruction.cxx@5465
> 
> problem: service writes in its destructor to data that has been already deleted (lives in the module)
> ```

**Comment by icecube-trac** [comment]
Trac update at 20190111T19:24:41: nwhitehorn commented:

> In changeset 168053:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="168053"
> Fixes #213.   Buggy test was written to chase down another bug in 2005, see 
> 
>   root-icetray/trunk/private/test/ServicesAtDestruction.cxx@5465
> 
> problem: service writes in its destructor to data that has been already deleted (lives in the module)
> ```
